### ISSUE TITLE
Purchase Order Roster in Excel file

### ISSUE THREAD

**Issue by pbrink231** [pbrink231/crm-issues#69]
While I am cleaning up sales orders should put in this temp fix.

We should have another tab on the import excel file.  This will have the roster.
- designcode
- size
- Roster Name (on garment)
- Roster Number (on garment)
- Contact (Name of person.  this will be grouped if by contact shipping)

**Comment by pbrink231** [comment]
Could also make the file an upload into the CRM so sales reps can do it immediately and get errors as they upload.

**Comment by pbrink231** [comment]
#37 Is about uploading excel file.

**Comment by pbrink231** [comment]
@rod-rosa311 Is this done?

**Comment by rod-rosa311** [comment]
@pbrink231 Yes, it has been added to manual POs along with the invoice number. Should be able to close it now .
